src/template/vite.config.js

```diff
@@ -1,7 +1,9 @@
 import { defineConfig } from "vite";
 
 export default defineConfig({
+    base: "./",
     build: {
         sourcemap: true,
+        outDir: "www",
```

Comment:
dist

Reply:
If we build into `dist` the `yarn zip` command won't find the correct directory:

```
"zip": "yarn run build && mkdir -p dist && zip -r dist/game.zip www -x \"**/.DS_Store\"",
```

```
$ yarn zip
✓ 5 modules transformed.
dist/index.html                  0.19 kB │ gzip:  0.17 kB
dist/assets/index-BNJisJFh.js  198.18 kB │ gzip: 73.23 kB
✓ built in 359ms
        zip warning: name not matched: www

zip error: Nothing to do! (dist/game.zip)
error Command failed with exit code 12.
```

Reply:
Happy to update both though